tests/postman.collection.Transmission.json

```diff
@@ -5,684 +5,1313 @@
+                      "{{transmission_url}}"
+                    ],
+                    "path": [
+                      "api",
+                      "v1",
+                      "shipments",
+                      "{{shipment_id}}",
+                      "tracking",
+                      ""
+                    ]
+                  }
+                },
+                "response": []
+              }
+            ]
+          },
+          {
+            "name": "AsyncJobs",
+            "item": [
+              {
+                "name": "List Jobs",
+                "request": {
+                  "method": "GET",
+                  "header": [
+                    {
+                      "key": "Authorization",
+                      "value": "JWT {{token}}"
+                    }
+                  ],
+                  "body": {
+
+                  },
+                  "url": {
+                    "raw": "{{transmission_url}}/api/v1/jobs/",
+                    "host": [
+                      "{{transmission_url}}"
+                    ],
+                    "path": [
+                      "api",
+                      "v1",
+                      "jobs",
+                      ""
+                    ]
+                  }
+                },
+                "response": [
+
+                ]
+              },
+              {
+                "name": "Get Job",
+                "request": {
+                  "method": "GET",
+                  "header": [
+                    {
+                      "key": "Authorization",
+                      "value": "JWT {{token}}"
+                    }
+                  ],
+                  "body": {
+
+                  },
+                  "url": {
+                    "raw": "{{transmission_url}}/api/v1/jobs/{{job_id}}/",
+                    "host": [
+                      "{{transmission_url}}"
+                    ],
+                    "path": [
+                      "api",
+                      "v1",
+                      "jobs",
+                      "{{job_id}}",
+                      ""
+                    ]
+                  }
+                },
+                "response": [
+
+                ]
+              },
+              {
+                "name": "Add Job Message",
+                "request": {
+                  "method": "POST",
+                  "header": [
+                    {
+                      "key": "Authorization",
+                      "value": "JWT {{token}}"
+                    }
+                  ],
+                  "body": {
+                    "mode": "formdata",
+                    "formdata": [
+                      {
+                        "key": "type",
+                        "value": "ERROR",
+                        "type": "text"
+                      },
+                      {
+                        "key": "body",
+                        "value": "{\"error\": \"wat hav u dun\"}",
```

Comment:
Check out the MessageSerializer - this isn't what the Job Message looks like. It'll look more like
```
{
  "type": "ERROR",
  "body": { "exception": "wat hav u dun"  }
}